### PR TITLE
Update quantum_repetition.yml

### PR DESCRIPTION
**Pull request by MazinKarjikar**
Added some repetition codes.

### CODE REVIEW

**Comment by MazinKarjikar** [comment]
I also added to the perfect codes file, added a realizations section to it, please let me know if you're satisfied with how I did it.

File: codes/quantum/qubits/quantum_repetition.yml [hunk 1]
```diff
@@ -35,6 +35,11 @@ realizations:
   - 'Liquid-state NMR: 3-qubit phase-flip code \cite{arxiv:quant-ph/9802018}.'
```
Comment (valbert4) [inline codes/quantum/qubits/quantum_repetition.yml:35]: 'Liquid-state NMR: 3-qubit phase-flip code \cite{arxiv:quant-ph/9802018,arXiv:1109.4821}.'

File: codes/quantum/qubits/quantum_repetition.yml [hunk 2]
```diff
@@ -35,6 +35,11 @@ realizations:
...
   - 'See Table S6 in Ref. \cite{arXiv:2102.06132} for a history of earlier implementations.'
   - 'Repetition codes can be used to benchmark device performance \cite{arxiv:2202.11045}.'
+  - 'Ion trap: 3-qubit phase-flip algorithm implemented in 3 cycles on high fidelity gate operations \cite{doi:10.1126/science.1203329}.'
+  - 'NMR: 3-qubit phase-flip code implemented in 2 cycles \cite{arXiv:1108.4842}.'
+  - 'Liquid-state NMR: 3-qubit phase-flip code \cite{arXiv:1109.4821}.'
```
Comment (valbert4) [inline codes/quantum/qubits/quantum_repetition.yml:40]: Delete this line, since I integrated it above.

**Comment by valbert4** [comment]
Thanks. Please merge the two NMR arxiv refs as I commented and resubmit.

**Pull request closed without merging**